parseProfiler.js

```diff
@@ -0,0 +1,37 @@
+export function parseProfiler(text, data) {
+  const obj = JSON.parse(text);
+  var processStartTime;
+  for (let row of data) {
+    if (row.operation == "Process Start") {
+      processStartTime = row.start
+      break;
+    }
+  }
```

Comment:
This can be expressed as a one-liner, which I think reads better?
```js
let processStartTime = data.find(row => row.operation == "Process Start").start;
```

Also, let's have `parseProfiler` take `processStartTime` in as an argument, instead of data, so it doesn't have to know anything about procmon's output. Then you can have it just return a new array, which we'll append to data. Something like:
```js
let processStartTime = data.find(row => row.operation == "Process Start").start;
data.push(...parseProfiler(profilerText, processStartTime));
```